packages/web-components/src/components/ic-select/ic-select.tsx

```diff
@@ -299,6 +367,11 @@ export class Select {
     this.searchable && this.handleFocusIndicatorDisplay();
   };
 
+  private handleMenuKeyPress = (ev: CustomEvent): void => {
+    const debounce = ev.detail.isNavKey === true ? 0 : this.debounce;
```

Comment:
```suggestion
    const debounce = ev.detail.isNavKey ? 0 : this.debounce;
```

Can we also add a comment above this so developers know it's to prevent lagging due to the debounce..